doc/decisions/0011-protocol-version-compatibility.md

```diff
@@ -0,0 +1,56 @@
+# 0011. Protocol compatibility between App versions
+
+| Date | Tags |
+|---|---|
+| Fri Oct 05  | protocol, messenger |
+
+## Status
+
+draft
+
+## Context
+
+This document enumerates the rules we have to adhere to when doing any protocol modifications and summarizes best practises to do so.
+In the light of recent issues (code landed in develop branch, producing "invisible" messages for previous app versions), it's best
```

Comment:
People won't know this recent issue, perhaps we can rewrite this as a form of "XX happened which lead to YY problem"